service/telemetry/internal/migration/v0.3.0.go

```diff
@@ -22,9 +22,8 @@ type TracesConfigV030 struct {
 	// tracecontext and  b3 are supported. By default, the value is set to empty list and
 	// context propagation is disabled.
 	Propagators []string `mapstructure:"propagators"`
```

Comment:
nit: independent of your change, we should replace this with https://pkg.go.dev/go.opentelemetry.io/contrib/otelconf/v0.3.0#Propagator

Reply:
correct, there was a pr opened by @jpkrohling to [address the issue](https://github.com/open-telemetry/opentelemetry-go-contrib/issues/6712), but it stalled